apps/indexer-coordinator/src/project/project.service.ts

```diff
@@ -215,6 +219,10 @@ export class ProjectService {
 
     const mmrPath = argv['mmrPath'].replace(/\/$/, '');
 
+    if (baseConfig.usePrimaryNetworkEndpoint !== false) {
```

Comment:
[suggestion] can be a share function in this service as there has 2 places have same code